subsys/bluetooth/controller/Kconfig

```diff
@@ -524,6 +525,11 @@ config BT_CTLR_ASSERT_HANDLER
 	  and will be invoked whenever the controller code encounters
 	  an unrecoverable error.
 
+config BT_CTLR_DF
+	bool "LE Direction Finding Support [EXPERIMENTAL]" if BT_LL_SW_SPLIT
```

Comment:
I'm not sure if we need to have `Support` in the prompt. I'm fine with keeping it as is

Reply:
I've removed `Support` to align with other LE XXX entries.